.github/workflows/build.yml

```diff
@@ -439,3 +439,39 @@ jobs:
 
           echo "Failed to install expected version $VERSION"
           exit 1
+
+  # finally, containerize the package and upload to the GHCR
+  build-and-push-to-ghcr:
+    needs: package
```

Comment:
Should we have a "latest" one too when code is pushed to dev?

Reply:
The job currently tags the latest container with `latest` and the version of the release. Are you saying it would add value to push 2 versions of a container: one reflecting the bleeding edge of the repository e.g. `latest-dev` and one reflecting the latest version on `PyPI` (`latest`)?

Reply:
We could have `latest` and the tagged versions be the same, ie: `4.2.0` and `latest`.

I like the idea of pushing one out when changes are committed to develop, but that one should be tagged something like `develop`, or `dev`. Thoughts?

Reply:
I can see this adding value to users who work exclusively in docker container environments for developing. Can do 👍

Reply:
No worries since you're already making the changes - But another possible way to stage these new changes could be in a new PR. Branch off this feature branch and then point it back at this feature branch with this specific change.

Reply:
Good point! Since I've already implemented the change, I won't do it this time but will do it in the future. I agree, I don't like big PRs :) I'm finishing up some troubleshooting work having to do with multiple untagged images being pushed alongside my tagged multi-platform images, after which I'll post new screenshots with more test results and re-request review. Thanks!